ros2cli/ros2cli/cli.py

```diff
@@ -29,6 +32,13 @@ def main(*, script_name='ros2', argv=None, description=None, extension=None):
         description=description,
         formatter_class=argparse.RawDescriptionHelpFormatter
     )
+    parser.add_argument(
```

Comment:
Nit: I prefer setting a `default=False` here.  I know that it is implied by `action='store_true'`, but it is just easier for the reader to know what is intended without reading the argparse documentation.

Reply:
Fixed in https://github.com/ros2/ros2cli/pull/659/commits/6c5a30e0ecc9ba90bfe27e1b924a236683318d8d.